src/controllers/ownerMessageController.js

```diff
@@ -15,7 +17,10 @@ const ownerMessageController = function (OwnerMessage) {
   };
 
   const updateOwnerMessage = async function (req, res) {
-    if (req.body.requestor.role !== 'Owner') {
+    
+    const canEditHeaderMessage = hasPermission(req.body.requestor, 'editHeaderMessage');
```

Comment:
```suggestion
    const canEditHeaderMessage = await hasPermission(req.body.requestor, 'editHeaderMessage');
```
Forgot hasPermission is async in my previous request.